next.config.ts

```
@@ -1,7 +1,9 @@
+import { withPayload } from "@payloadcms/next/withPayload";
+import { withPayload } from "@payloadcms/next/withPayload";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'withPayload'. Consider to delete it or rename it.

'withPayload' is defined here:


(lint/suspicious/noRedeclare)